src/bandwidth-tool/templates/app.vue

```diff
@@ -16,60 +16,70 @@ limitations under the License.
-                    :bandwidth-allowance-data="bandwidthAllowanceData"
-                    :bandwidth-consumption="bandwidthConsumption"
-                    :bandwidth-consumption-data="bandwidthConsumptionData"
-                    :bandwidth-overage="bandwidthOverage"
-                    :focused-droplet-class="focusedDropletClass"
-                    :focused-droplet-enter="focusedDropletEnter"
-                    :focused-droplet-leave="focusedDropletLeave"
-                ></Pool>
-            </template>
-            <template v-slot:buttons>
-            </template>
-        </Header>
+        <div class="container inset">
+            <div class="container">
+                <div class="panel bandwidth">
+                    <h1>{{ i18n.templates.app.title }}</h1>
+                    <h3><small>{{ i18n.templates.app.description }}</small></h3>
 
-        <div class="main container">
-            <h2>{{ i18n.templates.app.droplets }}</h2>
-            <div v-if="hasActiveDroplets">
-                <p class="has-text-muted">
-                    {{ i18n.templates.app.estimatedCost }} ${{ dropletCost.toLocaleString() }} / mo.
-                </p>
-                <div class="panel-list panel-list-vertical">
-                    <ActiveDroplet
-                        v-for="(droplet, id) in activeDroplets"
-                        :key="id"
-                        ref="activeDroplets"
-                        :droplet="droplet"
-                        :class="focusedDropletClass(id)"
-                        @mouseenter.native="focusedDropletEnter(id)"
-                        @mouseleave.native="focusedDropletLeave(id)"
-                        @remove="removed(id)"
-                        @update="update"
-                    ></ActiveDroplet>
-                </div>
-            </div>
-            <div v-else>
-                <p class="has-text-muted">
-                    {{ i18n.templates.app.selectToStart }}
-                </p>
-                <div class="panel-list panel-list-vertical">
-                    <SkeletonDroplet></SkeletonDroplet>
+                    <Pool
+                        :bandwidth-allowance="bandwidthAllowance"
+                        :bandwidth-allowance-data="bandwidthAllowanceData"
+                        :bandwidth-consumption="bandwidthConsumption"
+                        :bandwidth-consumption-data="bandwidthConsumptionData"
+                        :bandwidth-overage="bandwidthOverage"
+                        :focused-droplet-class="focusedDropletClass"
+                        :focused-droplet-enter="focusedDropletEnter"
+                        :focused-droplet-leave="focusedDropletLeave"
+                    ></Pool>
+
+                    <div class="droplets">
+                        <h2>{{ i18n.templates.app.droplets }}</h2>
+                        <div v-if="hasActiveDroplets">
+                            <!--<p class="has-text-muted">
```

Comment:
Is this comment meant to be here?

Reply:
Yeah, might need it in the future, app still calculates the value.